docs/releases/1.0.rst

```diff
@@ -84,9 +84,9 @@ Admin
  * Removed the dependency on ``LOGIN_URL`` and ``LOGIN_REDIRECT_URL`` settings
  * Password reset view names namespaced to wagtailadmin
  * Removed the need to add permission check on admin views (now automated)
- * Reversing ``django.contrib.auth.admin.login`` will no longer lead to Wagtails login view (making it easier to have front end views)
+ * Reversing ``django.contrib.auth.admin.login`` will no longer lead to Wagtails log in view (making it easier to have frontend log in views)
```

Comment:
Wagtails -> Wagtail's
In this particular case, should be "Wagtail's login view" as that is not an action.